### DISCUSSION TITLE
start_height=0

### DISCUSSION THREAD

**Discussion by Carl-arctan** [lightningnetwork/lnd#9094] [Q&A]
I plan to use Neutrino to open a node on mainnet. Here is the content of my lnd.conf file

```
[Application Options]
#debuglevel=info
debuglevel=trace
maxpendingchannels=5
alias=ExampleNode
color=#68F442
feeurl=https://nodes.lightning.computer/fees/v1/btc-fee-estimates.json

[Bitcoin]
bitcoin.mainnet=true
bitcoin.node=neutrino


[Neutrino]
neutrino.addpeer=btcd-mainnet.lightning.computer
#neutrino.addpeer=faucet.lightning.community
neutrino.addpeer=18.177.241.6:8333
```

Always stuck in “Waiting for chain backend to finish sync, start_height=0”. I also tried using trace and debug modes, which showed “Syncing to block timestamp: 2009-01-04 02:15:05 +0800 CST, is synced=false”. I tried many nodes but couldn't solve it.

**Comment by bitromortac** [comment]
Is this a fresh lnd wallet? Neutrino sync can sometimes be time consuming, could you add multiple peers that support [`NODE_COMPACT_FILTERS`](https://bitnodes.io/nodes/?q=NODE_COMPACT_FILTERS)?

**Reply by Carl-arctan** [reply]
Yes, it's a new wallet. I tried many connectable nodes found on [https://bitnodes.io/](), but none of them could synchronize.

**Reply by bitromortac** [reply]
What's the lnd version you are using?

**Reply by Carl-arctan** [reply]
I'm using LND version v0.18.2-beta (lnd-linux-amd64-v0.18.2-beta.tar.gz).

**Reply by bitromortac** [reply]
I used your configuration and added two peers from the link above and could sync without problems with lnd 0.18.2, for example:
```
[Neutrino]
neutrino.addpeer=91.49.95.84:8333
neutrino.addpeer=46.26.88.52:8333
```

Maybe your `.lnd` folder somehow has data that prevents syncing? If you've never used that node for anything, you could try with a fresh .lnd folder.

**Reply by Carl-arctan** [reply]
I tried deleting the` .lnd` folder and rebuilding, but it still didn’t work. I also tried turning off the firewall and testing on `testnet`, and the `testnet` was able to sync successfully.
